packages/api-explorer/src/form-components/URLWidget.jsx

```diff
@@ -0,0 +1,14 @@
+const extensions = require('@readme/oas-extensions');
+
+const URLWidget = require('react-jsonschema-form/lib/components/widgets/URLWidget').default;
+
+function createURLWidget(oas) {
+  const explorerEnabled = oas[extensions.EXPLORER_ENABLED];
+
+  // Return a function that renders null when the explorer is disabled
+  if (!explorerEnabled) return () => null;
```

Comment:
Can you add a test for this case like this one: https://github.com/readmeio/api-explorer/blob/master/packages/api-explorer/__tests__/Params.test.jsx#L212-L222

You may have to modify the fixture file to add a string with `format: url` to test this.